system_tests/common_test.go

```diff
@@ -281,10 +281,13 @@ func Fail(t *testing.T, printables ...interface{}) {
 	testhelpers.FailImpl(t, printables...)
 }
 
-func Create2ndNode(t *testing.T, ctx context.Context, first *arbnode.Node, l1stack *node.Node, l2InitData *statetransfer.ArbosInitializationInfo, blockValidator bool) (*ethclient.Client, *arbnode.Node) {
-	nodeConf := arbnode.ConfigDefaultL1Test()
-	nodeConf.BatchPoster.Enable = false
-	nodeConf.BlockValidator.Enable = blockValidator
+func Create2ndNode(t *testing.T, ctx context.Context, first *arbnode.Node, l1stack *node.Node, l2InitData *statetransfer.ArbosInitializationInfo, dasConfig *das.DataAvailabilityConfig) (*ethclient.Client, *arbnode.Node) {
+	nodeConf := arbnode.ConfigDefaultL1NonSequencerTest()
+	if dasConfig == nil {
+		nodeConf.DataAvailability.Enable = false
```

Comment:
Is there any use case for enabling or disabling data availability when it's required/not needed as part of the chain config? We might be able to just drop that option